README.md

```diff
@@ -39,6 +45,23 @@ class Filter {
    * extension in the list is replaced with the `targetExtension` option's value.
    */
   virtual getDestFilePath(relativePath: string): string;
+
+  /**
+   * Method `postProcess`: may be implemented on subclasses of
+   * Filter.
+   *
+   * This method can be used in subclasses to do processing on the results of
+   * each files `processString` method.
+   *
+   * A common scenario for this is linting plugins, where on initial build users
+   * expect to get console warnings for lint errors, but we do not want to re-lint
+   * each file on every boot (since most of them will be able to be served from the
+   * cache).
+   *
+   * The `.output` property of the return value is used as the emitted file contents.
+   */
+  postProcessedString(results: object, relativePath: string): object
```

Comment:
There is a mismatch between `postProcess` in the docs and `postProcessedString` in the code.

Can this return a string or an object like `processString`?

There is discrepancy between the docs here and on `processString` as to which output is actually used as the file contents.